### ISSUE TITLE
rendering problems

### ISSUE THREAD

**Issue by C4PTRAMPAGE** [Thutmose/Thut#145]
#### Issue Description:
when looking at the elevator weird rendering problems happen which include but are not limited to
![20190628162223_1](https://user-images.githubusercontent.com/36720741/60376025-03066780-99c1-11e9-918a-317ed9cabc00.jpg)
![20190628162226_1](https://user-images.githubusercontent.com/36720741/60376040-10bbed00-99c1-11e9-9994-26c1d3223830.jpg)
![20190628160916_1](https://user-images.githubusercontent.com/36720741/60376044-131e4700-99c1-11e9-8f93-f5ea7a41605a.jpg)
![20190628162229_1](https://user-images.githubusercontent.com/36720741/60376046-14e80a80-99c1-11e9-95f5-f0933bb26337.jpg)
depending on the angle you look at the elevator.

#### What happens:
this
![20190628160916_1](https://user-images.githubusercontent.com/36720741/60375906-62b04300-99c0-11e9-97b7-873da4708443.jpg)

#### What you expected to happen:
not this
![20190628160916_1](https://user-images.githubusercontent.com/36720741/60375934-896e7980-99c0-11e9-9957-51963e0ff8ab.jpg)

#### Steps to reproduce:
1. look at the elevator at a few different angles

#### Affected Versions (Do *not* use "latest"): Replace with a list of all mods you have in.
- Thutcore: 1.12.2-5.21.0
- Minecraft: 1.12.2
- Forge: 1.12.2-14.23.5.2838

**Comment by Thutmose** [comment]
Do you have any other mods installed? does it only happen when certain other blocks are nearby?

If you could make a small test world that demonstrates this, I can look into fixing it

**Comment by C4PTRAMPAGE** [comment]
yeah i can get you a test world... but i dont have access to that computer right now, and yes i'm running about 40 other mods but this didnt start happening until i got a more up to date version of your mod though iirc

**Comment by C4PTRAMPAGE** [comment]
ok it looks like something is causing any block that can fall (gravel / sand) to render like that while falling, so whatever it is its not exclusive to your mod.

**Comment by NiuQ2** [comment]
I know that this was quite a while ago now but I am having the same issue as you and was wondering if you found the mod that was causing the problems?

**Comment by C4PTRAMPAGE** [comment]
completely forgot i made this post... sorry no i haven't

**Comment by NiuQ2** [comment]
Ok thanks
